### ISSUE TITLE
no-unresolved does not check dynamic import

### ISSUE THREAD

**Issue by aladdin-add** [benmosher/eslint-plugin-import#2024]
```js
import("./component-xxx");
```

if the "./component-xxx" does not exists, the rule should report an error.

seems a reasonable enhencement?

**Comment by ljharb** [comment]
Absolutely, for string literals or template literals without interpolations.
